docs/src/add-services/mysql/_index.md

```diff
@@ -348,6 +348,31 @@ ALTER TABLE table_name CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_
 
 For further details, see the [MariaDB documentation](https://mariadb.com/kb/en/character-set-and-collation-overview/).
 
+## Password generation
+ 
+When you create an endpoint to allow your app and database to communicate,
+an empty password is generated by default.
+If you add multiple endpoints,
+having empty passwords can cause issues with your app.
```

Comment:
:lady_beetle: This seems to imply that you can add multiple endpoints and have empty passwords. That isn't something that can happen. It's not a problem that _users_ need to resolve, as implied in the next line.